src/miRBench/predictor.py

```
@@ -104,6 +111,40 @@ def __init__(self):
     def predict(self, data, **kwargs):
         return self.model.predict(data, **kwargs)
     
+class miRBenchCNN_Manakov(Predictor):
+    """
+    Based on Sammut, Gresova et al. "miRBench: novel benchmark datasets for microRNA binding site prediction that mitigate against prevalent microRNA frequency class bias." Bioinformatics, Volume 41, Issue Supplement_1, July 2025, Pages i542–i551. https://doi.org/10.1093/bioinformatics/btaf233.
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)